GetDKAN/dkan
# Fix the data story view to only display stories

Resource nodes and unpublished nodes are showing up on the /stories view

Replacing the search index view with simpler node type view.

### QA steps
- create a dataset with the title 'Fresh Produce Purchases - Quarterly Totals', see if it turn up on `/stories`
